templates/graphql-js-starter/package.json

```diff
@@ -5,10 +5,9 @@
   "private": true,
   "main": "dist/src/index.js",
   "scripts": {
-    "start": "ts-node src/index.ts",
+    "develop": "ts-node src/index.ts",
     "build": "tsc",
-    "serve": "node dist/src/index.js",
-    "watch": "graphback watch"
```

Comment:
watch can be still watching typescript code.` ts-node -w`. Then if you do graphback generate it will reload. This means we do not need to have graphback watch at all